bin/peggy-cli.mjs

```diff
@@ -285,10 +293,10 @@ export class PeggyCLI extends Command {
           }
         }
 
-        if (this.progOptions.test && this.progOptions.testFile) {
-          this.error("The -t/--test and -T/--test-file options are mutually exclusive.");
-        }
-        if (this.progOptions.test) {
+        if (typeof this.progOptions.test === "string") {
```

Comment:
I suppose that this check should fix #213? It is better to add a comment why exactly that check is required, or, in other words, why the `if (this.progOptions.test ...)` does not work

Reply:
Made it clear that this is to catch `-t ""`.

Reply:
Nice